### PR TITLE
Add PromMetrics to collect live data from a Prometheus Host

### PR DESCRIPTION
**Pull request by 4n4nd**
Added PromMetrics class which acts like an observble.
Added prometheus.py as a connector to a Prometheus Host.
Updated Pipfile to add apscheduler as a dependency.
Updated Pipfile.lock.

### CODE REVIEW

File: metrics/prommetrics.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            id='update_metric_data',
+            name='Ticker to get new data from prometheus',
+            replace_existing=True)
+
+        atexit.register(lambda: scheduler.shutdown()) # Shut down the scheduler when exiting the app
+
+    def _get_metrics_from_prometheus(self, observer):
+        # Collect credentials to connect to a prometheus instance
+        prom_token = os.getenv("PROM_ACCESS_TOKEN",None)
+        prom_url = os.getenv("PROM_URL",None)
```
Comment (42B) [inline metrics/prommetrics.py]:
```python
prom_token = os.getenv("PROM_ACCESS_TOKEN")
prom_url = os.getenv("PROM_URL")
```
?

File: metrics/prommetrics.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        if not (prom_token or prom_url):
+            print("Prometheus credentials not found")
+            exit(0)
```
Comment (42B) [inline metrics/prommetrics.py]: Shouldn't this use `sys.exit` and also return a nonzero value?

Comment (4n4nd) [reply metrics/prommetrics.py]: Good idea, I will change it.
Thanks

**Review by durandom — APPROVED** [review]
That looks great. 
Awesome work @4n4nd

**Pull request merged**